srcpkgs/python3-zeroconf

```diff
@@ -1 +0,0 @@
-python-zeroconf
```

Comment:
As `python-zeroconf` will replace `python3-zeroconf` , you can add to the template:
````
replaces="python3-zeroconf>=0"
provides="python3-zeroconf-${version}_${revision}"
````

Using these variables, when you upgrade `cura` you can see:
````
Name             Action    Version           New version            Download size
python3-zeroconf remove    0.19.1_2          -                      - 
python-zeroconf  install   -                 0.20.0_1               - 
cura             update    3.4.1_1           3.4.1_2                -
````

Without them, it's different:
````
Name            Action    Version           New version            Download size
python-zeroconf install   -                 0.20.0_1               - 
cura            update    3.4.1_1           3.4.1_2                -
````
The obsolete package `python3-zeroconf` will not be replaced.